### ISSUE TITLE
OVS switches interfaces get configured twice

### ISSUE THREAD

**Issue by edgarcosta92** [mininet/mininet#656]
I am using mininet 2.2.1.

I noted that OVS swtiches interfaces get configured twice. In my case I made a subclass of TCIntf and I added some Iptables rules at TCIntf.conf(). The problem is that the rules get added two times, I checked putting some prints and I realized that switches execute Intf.conf() two times : when adding links and when adding switches.

Kind regards,
Edgar

**Comment by lantz** [comment]
Yes, I think we should look at this. The root issue is that both OVS and Mininet can try to configure interfaces using `tc` and can conflict with each other.
